src/tokens-dist/css/variables.css

```diff
@@ -7,96 +7,205 @@
   --eds-z-index-200: 200; /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
   --eds-z-index-100: 100; /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
   --eds-z-index-0: 0; /* @deprecated This should not be used in code or design. It will be removed in a future version of EDS. */
-  --eds-size-2-and-half: calc(var(--eds-size-base-unit) * 2.5);
-  --eds-size-1-and-half: calc(var(--eds-size-base-unit) * 1.5);
-  --eds-size-quarter: calc(var(--eds-size-base-unit) / 4);
-  --eds-size-half: calc(var(--eds-size-base-unit) / 2);
+  --eds-size-2-and-half: 1.275rem;
```

Comment:
changing this back to the actual values as a first step to removing the CSS-specific values from the tokens.